Dockerfile

```diff
@@ -12,6 +12,7 @@ COPY build/libs/$APP /opt/app/
 
 WORKDIR /opt/app
 
-HEALTHCHECK --interval=10s --timeout=10s --retries=10 CMD http_proxy="" wget --spider -q http://localhost:8090/health || exit 1
+HEALTHCHECK --interval=10s --timeout=10s --retries=10 CMD http_proxy="" wget --spider -q http://localhost:8094/health || exit 1
```

Comment:
this breaks docker-compose snl utils, please revert to what it was.

Reply:
Reverted